src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar top-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--top"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar bottom-search-bar"
+      data-qa="small"
+    >
+      <input
+        class="search-bar-input search-bar-input--bottom"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
```

Comment:
Don't use spaces in links. Have you seen any link with literal space in it on the Internet? Remember, anchor links start with '#'.